adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/WEB-INF/tags/responsive/componentPayment.tag

```diff
@@ -0,0 +1,40 @@
+<%@ attribute name="paymentMethod" required="true" type="java.lang.String" %>
+
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags" %>
+
+<c:url value="/checkout/multi/adyen/summary/component-result" var="handleComponentResult"/>
+
+<c:choose>
+    <%-- Paypal button --%>
+    <c:when test="${selectedPaymentMethod eq 'paypal'}">
+        <div class="checkbox">
+            <label>
+                <input type="checkbox" id="Terms1" />
```

Comment:
typo? Terms1 instead of terms?

Reply:
Terms1 was the name already used by the other payment methods. I guess it might be an typo on the original code or the name "Terms" would clash with some other thing.
Changed the name to a more clear one.